deskpro/deskpro-admin-frontend
# Feature/6203 horizontal group rebased

Testing:

- Go to `/agents/settings`
- See bottom components are side-by-side in a horizontal configuration.
